nixos/modules/services/logging/journaldriver.nix

```diff
@@ -91,12 +91,12 @@ in
   config = mkIf cfg.enable {
     systemd.services.journaldriver = {
       description = "Stackdriver Logging journal forwarder";
-      script = "${pkgs.journaldriver}/bin/journaldriver";
       wants = [ "network-online.target" ];
       after = [ "network-online.target" ];
       wantedBy = [ "multi-user.target" ];
 
       serviceConfig = {
+        ExecStart = "${pkgs.journaldriver}/bin/journaldriver";
```

Comment:
This should use `lib.getExe` instead.

Reply:
I've gone through about 9 of them.

The point I was trying to make is that it's a bit painstaking to check the meta tag of every package one by one in the repl. Running `nix eval` won't provide me the warnings either, because the `lib.getExe` invocation remains unevaled behind `mkIf cfg.enable` in every module that does not have a nixos test. Worst case is that I accidentally `lib.getExe` a package without `mainProgram`, and the downstream users would get the warnings about missing `mainProgram`.

I imagine it would be possible to script such a treewide with a script that automatically checks for `meta.mainProgram`, and does a big find-replace. But it's not really in the scope of this PR, nor is it relevant for this PR's intended effect.